internal/service/privatelinkendpoint/resource_privatelink_endpoint_test.go

```diff
@@ -69,7 +69,7 @@ func TestAccNetworkRSPrivateLinkEndpointAWS_import(t *testing.T) {
 			},
 			{
 				ResourceName:      resourceName,
-				ImportStateIdFunc: testAccCheckMongoDBAtlasPrivateLinkEndpointImportStateIDFunc(resourceName),
+				ImportStateIdFunc: checkImportStateIDFunc(resourceName),
```

Comment:
```suggestion
				ImportStateIdFunc: importStateIDFunc(resourceName),
```
knitpit: in these case I'm using importStateIDFunc